File: .config/hiss/testdata/HISS-01/python/gap/mutual-recursion.py
def is_even(n):
    # is_even and is_odd form a two-function cycle. Deciding it needs a call graph across
    # functions, which the Python line scanner does not build.
    if n == 0:
        return True
    return is_odd(n - 1)


def is_odd(n):
    if n == 0:
        return False
    return is_even(n - 1)

File: .config/hiss/testdata/HISS-01/rust/gap/mutual-recursion.rs
// is_even and is_odd form a two-function cycle. Deciding it needs a call graph across
// functions, which the Rust line scanner does not build.
pub fn is_even(n: u32) -> bool {
    if n == 0 {
        return true;
    }
    is_odd(n - 1)
}

pub fn is_odd(n: u32) -> bool {
    if n == 0 {
        return false;
    }
    is_even(n - 1)
}
